cluster-operator/src/test/java/io/strimzi/operator/cluster/ResourceUtils.java

```diff
@@ -505,19 +506,29 @@ public static KafkaMirrorMaker createEmptyKafkaMirrorMakerCluster(String cluster
     }
 
     public static KafkaMirrorMaker createKafkaMirrorMakerCluster(String clusterCmNamespace, String clusterCmName, String image, KafkaMirrorMakerProducerSpec producer, KafkaMirrorMakerConsumerSpec consumer, String whitelist, Map<String, Object> metricsCm) {
-        return new KafkaMirrorMakerBuilder()
+        return createKafkaMirrorMakerCluster(clusterCmNamespace, clusterCmName, image, null, producer, consumer, whitelist, metricsCm);
+    }
+
+    public static KafkaMirrorMaker createKafkaMirrorMakerCluster(String clusterCmNamespace, String clusterCmName, String image, Integer replicas, KafkaMirrorMakerProducerSpec producer, KafkaMirrorMakerConsumerSpec consumer, String whitelist, Map<String, Object> metricsCm) {
+        KafkaMirrorMakerFluent.SpecNested<KafkaMirrorMakerBuilder> kafkaMirrorMakerBuilder = new KafkaMirrorMakerBuilder()
```

Comment:
I think this would work fine. But maybe it would be easier to read the code if you keep the one big blog with the Builder, build the `KafkaMirrorMaker`, then change the replicas and return it.